src/HomeWork.java

```diff
@@ -0,0 +1,59 @@
+public class HomeWork {
+    public static void initialize() {
+        byte byteValue = -110;
+        boolean booleanValue = true;
+        short shortValue = 1123;
+        long longValue = 100000L;
+        int intValue = 45677;
+        double doubleValue = 12.12;
+        float floatValue = 10000f;
+        char charValue = '*';
+    }
+
+    public static float CalcResult1(float a, float b, float c, float d) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.